src/components/ui/LoadingScreen.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import clsx from "clsx";
+
+export default function LoadingScreen() {
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    // Hide loading screen after the page has fully loaded
+    const handleLoad = () => {
+      setTimeout(() => {
+        setIsLoading(false);
+      }, 500); // Small delay to ensure smooth transition
+    };
+
+    // Check if document is already loaded
```

Comment:
```suggestion
    const handleLoad = () => {
      setTimeout(() => {
        setIsLoading(false);
      }, 500);
    };

```
Comments seem redundant to me